x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/engines/engines_list.tsx

```diff
@@ -57,18 +59,19 @@ export const CreateEngineButton: React.FC<{ disabled: boolean }> = ({ disabled }
           onFocus={() => setShowPopover(true)}
           tabIndex={0}
         >
-          <EuiButton
+          <EuiButtonTo
             fill
             iconType="plusInCircle"
             data-test-subj="enterprise-search-content-engines-creation-button"
             data-telemetry-id="entSearchContent-engines-list-createEngine"
-            disabled={disabled}
+            isDisabled={disabled}
             onClick={openEngineCreate}
+            to={ENGINE_CREATION_PATH}
```

Comment:
Replaced `EuiButton` to `EuiButtonTo`  in create search application buttons to help with redirecting  route to `/new` route path